common/common.scss

```diff
@@ -27,6 +27,91 @@
   }
 }
 
-.scheme-icon {
-  margin-right: 0.25em;
+// theme toggle button
+
+/* The switch - the box around the slider */
+#dark-light-toggle {
+  display: flex;
+  width: auto;
+
+  &:hover {
+    background-color: transparent;
+    .selected .d-icon {
+      color: transparent;
+    }
+  }
+}
+
+.switch {
+  position: relative;
+  width: 70px;
+  height: 35px;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+  border-radius: 35px;
+  border: 1px solid var(--primary-low-mid);
+  justify-content: space-between;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.switch span.toggle-icon {
+  display: flex;
+  align-items: center;
+  margin: 0 0.5em;
+  .d-icon {
+    width: 20px;
+    color: var(--primary-medium);
+    font-size: 20px;
+    align-self: center;
+  }
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  height: 18px;
+  width: 18px;
+  top: calc(50% - 12px);
+  left: 4px;
+  right: 0;
+  bottom: 0;
+  border-radius: 50%;
+  border: 3px solid var(--primary-low-mid);
+  -webkit-transition: transform 0.2s;
```

Comment:
`-webkit-transition` uses unprefixed property since 2015 so this is no longer neede